.cirrus.yml

```diff
@@ -5,12 +5,12 @@
 # See https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html
 
 windows_shard_template: &WINDOWS_SHARD_TEMPLATE
-  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') || $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
+  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') && $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
```

Comment:
we want this to ONLY run on post-submit?

Reply:
This was an error, back to ||